recipes/watchdog-wms/build.sh

```diff
@@ -0,0 +1,32 @@
+# stop on error
+set -eu -o pipefail
+
+# create output folder name
+VERSION="${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}"
+OUT="${PREFIX}/share/${VERSION}"
+
+# copy the files to /share/${VERSION}
+mkdir -p "${OUT}"
+mkdir -p "${PREFIX}/bin"
+cp -R * "${OUT}/"
+
+# rename the sh scripts
+mv "${OUT}/watchdog.sh" "${OUT}/watchdog-cmd.sh"
+mv "${OUT}/workflowDesigner.sh" "${OUT}/watchdog-gui.sh"
+
+# modify path of included files
+sed -i "s|\$SCRIPT_FOLDER/|\$SCRIPT_FOLDER/../share/${VERSION}/|g" "${OUT}/watchdog-cmd.sh"
```

Comment:
You'd have to triple-escape `\\\$SCRIPT_FOLDER` since then `bash` will pass on that part of the double quoted string as `\$SCRIPT_FOLDER` to `sed` which also needs the `$` to be escaped.

Reply:
Thanks for the hint! However, there was another problem with the macos version of sed as the -i parameter espects another argument.

Reply:
Ah, right, `sed` behaves differently/incompatible for inline modifications on Linux and OSX (i.e., the "OSX-style" `sed -i ''` won't work on Linux since there an additional argument is not valid :/). The only way to directly use it in a manner that works under both systems is to use something like `sed -i.bak` (i.e., create a backup file).

Reply:
I really should have noticed that before! Sorry 'bout that.

Reply:
No problem. Now it is passing the tests. :)